docs/userguide.md

```diff
@@ -72,32 +85,34 @@ If you would like to have the deployment of the federation-v2 control plane
 automated, then invoke the deployment script by running:
 
 ```bash
-./scripts/deploy-federation-latest.sh cluster2
+./scripts/deploy-federation-latest.sh ${MEMBER_CLUSTER}
 ```
 
 The above script joins the host cluster to the federation control plane it deploys, by default.
 The argument(s) used is/are the list of context names of the additional clusters that needs to be
-joined to this federation control plane. Clarifying, say the `host-cluster-context` used is `cluster1`,
-then on successful completion of the script used in example, both `cluster1` and `cluster2` will be
+joined to this federation control plane. Clarifying, say the
+`host-cluster-context` used is `HOST_CLUSTER`,
+then on successful completion of the script used in example, both
+`HOST_CLUSTER` and `MEMBER_CLUSTER` will be
 joined to the deployed federation control plane.
 
 **NOTE:** You can list multiple joining cluster names in the above command.
 Also, please make sure the joining cluster name(s) provided matches the joining
 cluster context from your kubeconfig. This will already be the case if you used
-the minikube instructions above to create your clusters.
+the kind instructions above to create your clusters.
 
 ## Operations
 
 ### Join Clusters
 
-Next, you'll want to use the `kubefed2` tool to join all your
-clusters that you want to test against.
+If you want to join your clusters manually, you'll want to use the `kubefed2`
+tool to join all your clusters that you want to test against.
 
 ```bash
-./bin/kubefed2 join cluster1 --cluster-context cluster1 \
-    --host-cluster-context cluster1 --add-to-registry --v=2
-./bin/kubefed2 join cluster2 --cluster-context cluster2 \
-    --host-cluster-context cluster1 --add-to-registry --v=2
+for c in ${CLUSTERS}: do
```

Comment:
(No action required) I think multi-line bash in a tutorial could be problematic, as pasting multi-line code can be a fraught exercise.  I notice that was already the practice prior to this change, so I'm not suggesting a change for this PR, but it may be worth considering whether this practice should remain or a switch should be made to more novice-friendly scripting.